README.md

```diff
@@ -0,0 +1,13 @@
+# Dummy Rollup Deploy
+
+## Local Dev
+
+```
+$ docker compose up
+```
+
+If you see this error:
+```
+error acting as staker                   err="error advancing stake from node 2 (hash 0xee288c5dcc61206e6868fa7a01da6abaabe22d6c849718a47eb361857b7e8dd8): error generating node action: block validation is still pending"
+```
+Let it sit for a bit.
```

Comment:
I think instead of saying this, we should say this error means that there are no new nodes to stake ( or no new batches that have been posted since the last batch)